docs/extensions/community.md

```diff
@@ -1,6 +1,9 @@
 # Known Extensions List
+
 This file should have all the known Plates extensions listed.
 
+Please feel free submit a [Pull Request](https://github.com/thephpleague/plates/compare) to add your extension to this list.
+
  - [Laravel Provider](https://github.com/franzliedke/laravel-plates)
  - [Attributes Rendering](https://github.com/RobinDev/platesAttributes)
  - [Includer](https://github.com/odahcam/plates-includer) 
```

Comment:
I think we'll need a short description for these as well since it's not obvious what Includer is or Attributes Rendering.

The laravel provider is ok, since it's clear that the package is a laravel service provider.

Reply:
done.